Library/Tracking/KSRAnalytics.swift

```diff
@@ -1440,6 +1440,18 @@ public final class KSRAnalytics {
     )
   }
 
+  /// Call when a users clicks Read more about the campaign button.
+  /// - Parameter project: The project the
+  public func trackCampaignDetailsButtonClicked(project: Project) {
```

Comment:
Incomplete.

Reply:
This is the incorrect format for the documentation. Look at the function above this one for a reference.